### ISSUE TITLE
setDisplayname function

### ISSUE THREAD

**Issue by Unshown** [Starystars67/BeamNG-MP#624]
A function in BeamMP to change the nickname of a user to be able to hide potential server staff from being recognized as staff.

This would allow for custom naming scemes for certain servers besides just having the roles feature.

**Comment by Starystars67** [comment]
The only issue I see here is that this also would allow for impersonation of other users.

**Comment by Unshown** [comment]
> The only issue I see here is that this also would allow for impersonation of other users.

Thats a good point. Would be up to the server owner to use it properly on his server i sppose

**Comment by OfficialLambdax** [comment]
@WiserTixx Could this be brought up in the next meeting? Question is, do you we want to support this or nah. Should be easy enough to implement

**Comment by WiserTixx** [comment]
I don't think that this is something that we want to support. Users can easily tell if a username is real by going to the forum, so if server staff want to mask themselves it would be better to just make an alternate account. It would also most likely mess up server plugins that communicate actions with client mods by using usernames.
